lib/src/widgets/switch/ds_switch_tile.widget.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DsSwitchTile extends StatelessWidget {
+  /// Create a tile widget with a switch button
+  ///
+  ///It is mandatory to inform three parameters: [value], enabling or disabling the switch, [onChanged], the event that will be executed when changing the value, and [title], which is the title of the tile.
+  const DsSwitchTile({
+    Key? key,
+    required this.value,
+    required this.onChanged,
+    required this.title,
+    this.switchScale = 1.0,
```

Comment:
I don't think the switch size should be modified. You should talk with Torugo, but I guess the size is always the same.

Reply:
The size is an existing customization in the widget. This parameter was included to make it more flexible and reusable. I don't think it interferes with this issue, as long as we put the correct scale. The default is 1.0. If it's still an impediment, I don't see a problem fixing it.